src/Cli/ConfigGenerator.cs

```diff
@@ -1775,24 +1775,38 @@ public static bool TryAddTelemetry(AddTelemetryOptions options, FileSystemRuntim
                 return false;
             }
 
-            if (string.IsNullOrWhiteSpace(options.AppInsightsConnString))
+            if (options.AppInsightsEnabled is CliBool.True && string.IsNullOrWhiteSpace(options.AppInsightsConnString))
             {
                 _logger.LogError("Invalid Application Insights connection string provided.");
                 return false;
             }
 
+            if (options.OpenTelemetryEnabled is CliBool.True && string.IsNullOrWhiteSpace(options.OpenTelemetryEndpoint))
+            {
+                _logger.LogError("Invalid OTEL endpoint provided.");
+                return false;
+            }
+
             ApplicationInsightsOptions applicationInsightsOptions = new(
                 Enabled: options.AppInsightsEnabled is CliBool.True ? true : false,
                 ConnectionString: options.AppInsightsConnString
             );
 
+            OpenTelemetryOptions openTelemetryOptions = new(
```

Comment:
Can you add some tests in `Cli.Tests/AddTelemetryTests.cs` to verify changes cli commands are able to update the config correctly?